### PR TITLE
Day15

### PR DESCRIPTION
**Pull request by wavrin**

### CODE REVIEW

**Comment by kotp** [comment]
Curious about that.  I thought bundler helped manage versions of Gems, not versions of Ruby.  How are you managing your version(s) of Ruby?  Or do you have a single version of Ruby?

**Comment by wavrin** [comment]
Sorry, I wasn't very clear in that. ;) You are correct about bundler. I'm using one version of Ruby (via homebrew).

File: 2_adventures/003/wavrin/15/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Today I thought a lot about managing ruby versions. I'd recently upgraded computers and wanted to change how I managed ruby versions.
+
+On previous computer I used rvm + gemsets. But that seemed to get a bit hairy.
+
+On new computer, I'm using just new version of ruby and letting Bundler handle gem versions.
```
Comment (adeluccar) [inline 2_adventures/003/wavrin/15/README.md:5]: If you use homebrew, `ruby-install` is a great utility and works quite well with `chruby` to provide multiple rubies on your machine (installed in ~/.rubies/ so they're easy to nuke).

File: 2_adventures/003/wavrin/15/README.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+Today I thought a lot about managing ruby versions. I'd recently upgraded computers and wanted to change how I managed ruby versions.
+
+On previous computer I used rvm + gemsets. But that seemed to get a bit hairy.
```
Comment (adeluccar) [inline 2_adventures/003/wavrin/15/README.md:3]: That's why I stopped using RVM. It felt bloaty and messed with too many things.

File: 2_adventures/003/wavrin/15/README.md (hunk 1 above)
Comment (wavrin) [inline 2_adventures/003/wavrin/15/README.md:5]: Thanks for that advice!

**Comment by patmaddox** [comment]
Go you!

**Comment by patmaddox** [comment]
I use rbenv with its [ruby-build plugin](https://github.com/sstephenson/ruby-build)

**Pull request merged**